donate/templates/main.html

```diff
@@ -122,7 +122,7 @@ <h3>Make a donation</h3>
                     <!-- Donor information -->
                     <div class="form-group">
                       <label for="donor[email]">Email</label>
-                      <input type="email" class="form-control" name="donor[email]" placeholder="edward.snowden@eff.org" />
+                      <input type="email" class="form-control" name="donor[email]" placeholder="edward.snowden@eff.org" required />
```

Comment:
Does adding required here prevent form submission when the user clicks `I wish to be Anonymous`